packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -266,7 +266,7 @@ describe("IntervalCollection fuzz testing", () => {
 		// TODO:AB#5337: re-enable these seeds.
 		skip: [
```

Comment:
did you check that all of these are repros of 0x54e? I do remember it being more common than the RB tree crash, but not sure it was all of them.

Reply:
Yeah all tests that exercise the RB tree crash also seem to exercise the 0x54e crash. I did forget to update the comments here, so I've made that change.